systemtest/loadgen/eventhandler.go

```diff
@@ -33,15 +33,23 @@ import (
 //go:embed events/*.ndjson
 var events embed.FS
 
+type EventHandlerParams struct {
+	Path    string
+	Url     string
```

Comment:
```suggestion
	URL     string
```

Nit: let's please follow https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
Oh thansk, will update 👍